fixtures/common/exercises/track-and-field/metadata.yml

```diff
@@ -0,0 +1,3 @@
+---
+blurb: "This is a track hint."
+source: "The internet."
```

Comment:
Certainty a wise choice, but in this case the changes affect only the *track* so you can use an existing problem without any trouble.